druid-shell/src/platform/x11/window.rs

```diff
@@ -562,16 +511,11 @@ impl Window {
             button,
             wheel_delta: Vec2::ZERO,
         };
-        match self.handler.try_borrow_mut() {
-            Ok(mut handler) => {
-                handler.mouse_down(&mouse_event);
-                Ok(())
-            }
-            Err(err) => Err(Error::BorrowError(format!(
-                "Window::handle_button_press handle: {}",
-                err
-            ))),
-        }
+        self.handler
+            .try_borrow_mut()
+            .context("Window::handle_button_press")?
```

Comment:
Lets keep the variable name in the context message. (`handler` instead of `handle` though)